rotkehlchen/chain/ethereum/modules/sushiswap/constants.py

```diff
@@ -0,0 +1,4 @@
+from typing import Final
+
+CPT_SUSHISWAP_V1: Final = 'sushiswap-v1'
```

Comment:
Nitpick/question: Shouldn't you also qualify the Final as str? Or does it auto detect it thanks to the assignent?

Answering my own question.
According to the [PEP](https://peps.python.org/pep-0591/#id2) it should:

> The typechecker should apply its usual type inference mechanisms to determine the type of ID (here, likely, int). Note that unlike for generic classes this is not the same as Final[Any].

Also a quick test seems to show this

```python
from typing import TYPE_CHECKING, Final

GLOBAL1: Final = 'foo'
GLOBAL2: Final = 55
GLOBAL3: Final = 22.11
GLOBAL4: Final = {'a': 1}

if TYPE_CHECKING:
    reveal_type(GLOBAL1)
    reveal_type(GLOBAL2)
    reveal_type(GLOBAL3)
    reveal_type(GLOBAL4)
```

and then `mypy test.py`

```
test4.py:13: note: Revealed type is "Literal['foo']?"
test4.py:14: note: Revealed type is "Literal[55]?"
test4.py:15: note: Revealed type is "builtins.float"
test4.py:16: note: Revealed type is "builtins.dict[builtins.str, builtins.int]"
Success: no issues found in 1 source file
```